src/components/button/button.stories.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react'
-import { Button } from './button'
+import { Button } from './extensions-button'
```

Comment:
Had to rename the button file to avoid style bleeding from auto-generated classnames from linaria